src/main/java/com/netchill/services/streaming/StreamingService.java

```diff
@@ -12,31 +12,41 @@
 @Singleton
 public class StreamingService {
     private MovieDbApiClient movieDbApiClient;
```

Comment:
En read only ici, le champs ne dois pas être modifiable hors du constructeur.
```suggestion
    private final MovieDbApiClient movieDbApiClient;
```